app/components/elements/form/error/base.js

```diff
@@ -0,0 +1,2 @@
+export default ({error, className = '', ...props}) =>
+  error && <span className={`${className} field-error`}>{error}</span>
```

Comment:
Should use ternary here otherwise `error` can be printed in the DOM if `0`
Or check for type